### DISCUSSION TITLE
Show linked repositories in projects view

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#62451] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

We're using 1 repository with 1 project and therefore we often jump between those.
In the repository I can easily navigate to the linked project but not vice versa.
Therefore I suggest a similar solution for the projects view: add a tab/button etc. which then shows all linked repositories

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ClaudioFriederich** [comment]
See https://github.com/orgs/community/discussions/41797 for further thoughts on this issue.
